include/dsHost.h

```diff
@@ -137,46 +137,89 @@ dsError_t dsHostTerm();
  */
 dsError_t dsGetCPUTemperature(float *cpuTemperature);
 
+/**
+ * @brief Returns the SOC ID Length
+ *
+ * @param[out] pSocIDLength - Maximum SOC ID length
+ *
+ * @return dsError_t                        - Status
+ * @retval dsERR_NONE                       - Success
+ * @retval dsERR_NOT_INITIALIZED            - Module is not initialised
+ * @retval dsERR_INVALID_PARAM              - Parameter passed to this function is invalid
+ * @retval dsERR_OPERATION_NOT_SUPPORTED    - The attempted operation is not supported
```

Comment:
under what condition would it be not supported? can't see this as a valid return code?